drivers/kscan/kscan_handlers.c

```diff
@@ -32,3 +32,35 @@ static int z_vrfy_kscan_enable_callback(const struct device *dev)
 	return z_impl_kscan_enable_callback((const struct device *)dev);
 }
 #include <syscalls/kscan_enable_callback_mrsh.c>
+
+static int z_vrfy_kscan_config_matrix(const struct device *dev, struct kscan_matrix_config *config)
```

Comment:
Should all these APIs use `kscan_matrix` as the prefix?

```
kscan_matrix_config()
kscan_matrix_drive_column()
kscan_matrix_read_row()
kscan_matrix_resume_detection()
```

Reply:
Done.